new/prusti/tests/pass/parse/composite.rs

```diff
@@ -8,49 +7,49 @@
 
 use prusti_contracts::*;
 
-#[requires(true && true ==> true)]
+#[requires(true && true ==> true && true)]
 fn test1() {}
 
-#[requires(true && (true ==> true))]
+#[requires(true && (true ==> true) && (true || true) && true)]
 fn test2() {}
 
-#[requires((true && true) ==> true)]
+#[requires((true && true) ==> true && true && true)]
 fn test3() {}
 
-#[requires((true ==> true) && true ==> true)]
+#[requires((true ==> true) && true ==> true && true && true && true)]
 fn test4() {}
 
-#[requires((true ==> true) && (true ==> true))]
+#[requires((true ==> true) && (true ==> true && (true || true)) && true && true && true && true)]
 fn test5() {}
 
-#[requires((true && true) ==> true ==> true)]
+#[requires((true && true) ==> true ==> true ==> true ==> true && true && true && true && true && true)]
 fn test6() {}
 
-#[requires((true && true) ==> (true && true))]
+#[requires((true && true) ==> (true && true) ==> (true && true) && true && true && true && true && true && true)]
 fn test7() {}
 
-#[requires((true || true) ==> (true || true))]
+#[requires((true || true) ==> (true || true) && true && true && true && true && true && true && true)]
 fn test8() {}
 
-#[requires((true || true) ==> (true || (true && true)))]
+#[requires((true || true) ==> (true || (true && (true || true))) && true && true && true && true && true && true && true && true)]
 fn test9() {}
 
-#[requires(true && forall(|a: i32| a == 5) && true)]
+#[requires(true && forall(|a: i32| a == 5) && true && true && true && true && true && true && true && true && true && true)]
 fn test10() {}
 
-#[requires(true && forall(|a: i32| a == 5))]
+#[requires(true && forall(|a: i32| a == 5) && true && true && true && true && true && true && true && true && true && true)]
 fn test11() {}
 
-#[requires(forall(|a: i32| a == 5) && true)]
+#[requires(forall(|a: i32| a == 5) && true && true && true && true && true && true && true && true && true && true && true && true)]
 fn test12() {}
 
-#[requires(true ==> forall(|a: i32| a == 5) ==> true)]
+#[requires(true ==> forall(|a: i32| a == 5) ==> true && true && true && true && true && true && true && true && true && true && true && true && true)]
```

Comment:
This looks pretty terrible. Could you please add code that removes UUIDs before sorting? It would be best to put it under a command-line option.

Reply:
i will, what do you mean by "It would be best to put it under a command-line option."? like i need to run `cargo test --some-option`?

Reply:
At the beginning of the test file, you have a special comment that contains the additional command-line options:
```rust
 // compile-flags: -Zprint-desugared-specs -Zprint-typeckd-specs -Zskip-verify
```

These are parsed [here](https://github.com/viperproject/prusti-dev/blob/296891d28e007edf7ce1f4f5701aa382b3378cf8/new/prusti/src/bin/prusti-driver.rs#L35-L49)